bisect.py

```diff
@@ -0,0 +1,12 @@
+import bisect
+
+my_list = [11, 25, 36, 47, 56, 69, 69, 69, 78, 78, 91, 102, 120]
+print('Correct Location to insert 53 is: ' + str(bisect.bisect(my_list, 53, 0, len(my_list))))
```

Comment:
Hey @FazilUdupi,
I tried running your script and it ran successfully, but **LGTM** is showing following error on **Line 4**:
```python
TypeError: 'module' object is not callable
```
It would be good if you could resolve this.
Thank you.